Update health.mdx (#16645)
hashicorp/vault: website/content/api-docs/system/health.mdx

  "cluster_id": "00af5aa8-c87d-b5fc-e82e-97cd8dfaf731"
}
```

### Sample Request to customize the status code being returned

```shell-session
$ curl -i https://127.0.0.1:8200/v1/sys/health\?drsecondarycode\=200

HTTP/2 200
cache-control: no-store
content-type: application/json
strict-transport-security: max-age=31536000; includeSubDomains
content-length: 364
date: Wed, 26 Jan 2022 09:21:13 GMT
```

### Sample Response

This response is only returned for a `GET` request.

Note: `replication_performance_mode` and `replication_dr_mode` reflect the state of
the active node in the cluster; if you are querying it for a standby that has

> just come up, it can take a small time for the active node to inform the

Reviewer: ```suggestion
just come up, it may take time for the active node to inform the
```
